### PR TITLE
Added doc about generation Azure credentials for Brizo

### PR DESCRIPTION
**Pull request by jcortejoso**
## Description

Added documentation about how to generate the required credentials for Brizo/Osmosis in Azure, using Azure web interface.

### CODE REVIEW

**Comment by ttmc** [comment]
Discussed offline.

**Comment by ttmc** [comment]
I will make the discussed changes to this pull request today.

**Review by ttmc — APPROVED** [review]

File: doc/devops/brizo-azure-credentials.md [hunk 1]
```diff
@@ -1,59 +1,161 @@
-# Create Azure Credentials for Brizo
+---
+title: Create Azure Credentials for Brizo
+description: Tutorial about how to set up Azure storage for use with Brizo
+slug: /tutorials/azure-for-brizo/
+section: concepts
```
Comment (kremalicious) [inline doc/devops/brizo-azure-credentials.md]: section should be `tutorials`

Comment (ttmc) [reply doc/devops/brizo-azure-credentials.md]: Doh!

**Pull request merged**